dev-lang/zig/zig-0.4.0.ebuild

```diff
@@ -0,0 +1,50 @@
+# Copyright 2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+inherit cmake-utils
+
+DESCRIPTION="A robust, optimal, and maintainable programming language."
+HOMEPAGE="https://ziglang.org/"
+LICENSE="MIT"
+SLOT="0"
+IUSE=""
+
+if [[ ${PV} == 9999 ]]; then
+	EGIT_REPO_URI="https://github.com/ziglang/zig.git"
+	inherit git-r3
+else
+	SRC_URI="https://github.com/ziglang/zig/archive/${PV}.tar.gz -> zig-${PV}.tar.gz"
+	KEYWORDS="~amd64 ~x86"
```

Comment:
Did you test this on x86? Asking due to complicated llvm dep.

Reply:
No, i figured it would work on x86 if it works on amd64